include/hip/hcc_detail/hip_fp16_math_fwd.h

```diff
@@ -80,3 +80,4 @@ extern "C"
     __device__ __attribute__((const)) __2f16 __ocml_sqrt_2f16(__2f16);
     __device__ __attribute__((const)) __2f16 __ocml_trunc_2f16(__2f16);
 }
+#endif // __CLANG_HIP_RUNTIME_WRPPER_INCLUDED__
```

Comment:
`// !__CLANG_HIP_RUNTIME_WRPPER_INCLUDED__`

Reply:
fixed

Reply:
Here I meant adding `!` in all "#endif" comments to match the corresponding `#if`.

Reply:
sorry. this time really fixed